config/examples/message_bus.yml

```diff
@@ -7,20 +7,23 @@ production:
     url: <%= ENV.fetch('MESSAGE_BUS_REDIS_URL', 'redis://localhost/8') %>
     pool_size: <%= ENV.fetch('RAILS_MAX_THREADS', 5) %>
     pool_timeout: 5
+    namespace: system-mbus
 
 preview:
   <<: *default
   redis:
     url: <%= ENV.fetch('MESSAGE_BUS_REDIS_URL', 'redis://localhost/8') %>
     pool_size: <%= ENV.fetch('RAILS_MAX_THREADS', 5) %>
     pool_timeout: 5
+    namespace: system-mbus
```

Comment:
I propose not to use the namespace for default configuration
i.e. Leave `namespace` blank

Reply:
Only in preview or always, @hallelujah ?

Reply:
See https://github.com/3scale/porta/pull/728/commits/05682c335eb6f42e02b539adee02f275ff3d1e87

We should not break existing configuration. Make it configurable ok, but former config should work.